src/main/java/org/aludratest/service/gui/web/selenium/selenium2/Selenium2Wrapper.java

```diff
@@ -1475,8 +1475,10 @@ public void deleteCookieNamed(String name) {
     	driver.manage().deleteCookieNamed(name);        
     }
 
-    public void addCookie(String name, String value, String domain, String path, Date expiry) {
-    	driver.manage().addCookie(new Cookie(name, value, domain, path, expiry));        
+    public void addCookie(String name, String value, String domain, String path, int expiry) {
+        Calendar cal = Calendar.getInstance();
+        cal.add(Calendar.SECOND, expiry);
+    	driver.manage().addCookie(new Cookie(name, value, domain, path, cal.getTime()));        

```

Comment:
This implementation requires a caller to specify an explicit expiry time (sorry, did not see that in the interface specs). Would appreciate accepting -1 value for expiry, leading to null being passed as Date to the Cookie constructor here (which means no expiry date set according to Javadoc).